config/initializers/redis.rb

```diff
@@ -5,7 +5,7 @@ class RedisSetting
 
   def initialize(config = nil)
     @config = {
-      url: ENV.fetch('REDIS_URL', nil)
+      url: ENV.fetch('REDIS_CACHE_URL', nil)
```

Comment:
I think this should probably fallback to the worker env as we store a few things which we won't want to be cleared every time we decide to clear the cache

Reply:
@darokel Since [L21](https://github.com/DFE-Digital/publish-teacher-training/blob/1f4c56af16cf99f630982b4f43438725f721a45d/config/initializers/redis.rb#L21-L22) is looking for `cache` in `VCAP_SERVICES` should we not use that? Cache for cache? I'll be guided by you of course :)

Reply:
Are the changes in the `docker-compose.yml` and `cable.yml` OK as well @darokel @defong ?

Reply:
In register we split the two. My mistake was thinking that Publish was setup in the same way. I think we wanted to use one for cache and one for non clearable stuff but it looks like it's always been using the cache (which shouldn't have been the case and explains why we had to reset the feature flags when we had to clear stuff in the past). I'd rather we set it up correctly in AKS

Reply:
Updated to `REDIS_WORKER_URL`